build-support/bin/get_os.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
+
+# Outputs the current operating system and variant, as used by BinaryUtils.
+# Example output: mac/10.13
+# Example output: linux/x86_64
+
```

Comment:
Why a forked subprocess rather than a function call? Just wondering if there are best practices I'm not aware of.

Reply:
Moving it to its own file made it easier for me to re-use its contents in a standalone way in the other place I want to use it.

I could reasonably keep it as a function, and write:
```
source get_os.sh
get_os
```

but just calling `./get_os.sh`

seemed cleaner/simpler.

Happy to switch it back to a function if you'd prefer, but I'd still want it to be in a standalone file in the same directory as download_binary.sh (so it's easy to locate, and isolate), rather than in a file with a bunch of other stuff in the directory above...

Reply:
I don't feel strongly, and I don't have any concrete reasons. Just a feeling about library code existing as functions.